packages/i18n/src/locales/en/options/bella.yml

```diff
@@ -10,6 +10,10 @@ bustSpanEase:
   title: Bust span ease
   description: Controls the amount of (horizontal) ease added to your bust span when locating the bust point.
 
+shoulderToShoulderEase:
```

Comment:
Looks like this needs to be added to all locales. I believe that is why the build is failing. Added a commit.